File: Python/1000-1999/1176.py
# -*- coding: utf-8 -*-

def main():
    n = int(input())
    fibonachi = [0, 1, 1, 2]
    

    for c in range(0, n):
        f = int(input())
        fib1 = 1
        fib2 = 1
        fib3 = 1

        if f <= 3:
            print('Fib(%i) = %i' %(f, fibonachi[f]))
        
        else:
            for c in range(2, f):
                fib1 = fib2
                fib2 = fib3
                fib3 = fib1 + fib2
            
            print('Fib(%i) = %i' %(f, fib3))


main()

File: C/1000-1999/1176.c
#include <stdio.h>

int main(void) {
    int num, vez;
    double fib1, fib2, nvfib;
    scanf("%d", &num);
    for (int c = 0; c < num; c++){
        scanf("%d", &vez);
        fib1 = 0;
        fib2 = 1;
        nvfib = 1;
        for (int i = 0; i < vez; i++){
            fib1 = fib2;
            fib2 = nvfib;
            nvfib = fib1 + fib2;
        }
        printf("Fib(%d) = %.0lf\n", vez, fib1);

    }
    return 0;
}
